itkwidgets/viewer.py

```diff
@@ -230,11 +230,11 @@ def set_axes_enabled(self, enabled: bool):
     def set_background_color(self, bgColor: List[float]):
         self.queue_request('setBackgroundColor', bgColor)
 
-    def set_image(self, image: Image):
+    def set_image(self, image: Image, name: str = 'image'):
```

Comment:
`'image'` -> `'Image'` to match the default

Reply:
Done